doc/Contribution.md

```diff
@@ -0,0 +1,106 @@
+# Contribution Guidelines
+==================
+## Overview
+Being an Open Source project, everyone can contribute, provided that it respect the following points:
+* Before contributing any code, the author must make sure all the tests work (see below how to launch the tests).
+* Developed code must adhere to the syntax guidelines enforced by the linters.
+* Code must be developed following the branching model and changelog policies defined below.
+* For any new feature added, unit tests must be provided, following the example of the ones already created.
+
+In order to start contributing:
+1. Fork this repository clicking on the "Fork" button on the upper-right area of the page.
```

Comment:
I think a blank line is missing for a proper visualization, please check: https://github.com/telefonicaid/iotagent-node-lib/blob/task/contributionPolicies/doc/Contribution.md